plugins/view-resources/src/components/ActionsPopup.svelte

```diff
@@ -241,16 +254,32 @@
 
   let items: SearchActionItem[] = []
 
-  async function updateItems (query: string, filteredActions: Array<WithLookup<Action>>): Promise<void> {
+  const updateItems = reduceCalls(async (query: string, filteredActions: Array<WithLookup<Action>>): Promise<void> => {
     let searchItems: SearchItem[] = []
     if (query !== '' && query.indexOf('/') !== 0) {
       searchItems = (await searchFor('spotlight', query)).items
     }
     items = packSearchAndActions(searchItems, filteredActions)
-  }
+  })
 
   $: void updateItems(search, filteredActions)
 
+  function txListener (tx: Tx): void {
```

Comment:
Nice change! There is a logic in `indexer.spec.ts` that reopens the popup in case of no matches, we should probably remove it to cover this change with test.

Reply:
Let's do it next time.